README.md

```diff
@@ -6,6 +6,73 @@
     </a>
 </p>
 
+## Upgrading Data from Sulu 2.6 to Sulu 3.0
+
+> ![Note]
+> The upgrade from Sulu 2.6 to Sulu 3.0 is a major upgrade and will require some migration steps.
+
+You can find the full upgrade guide here: [sulu/sulu UPGRADE.md](https://github.com/sulu/sulu/blob/3.0/UPGRADE-3.x.md)
+
+## How to install
+
+Install the package:
+
+```shell
+composer require sulu/phpcr-migration-bundle
+```
+
+Add the new migration bundle to your `config/bundles.php`:
+
+```diff
+// config/bundles.php
+
+return [
+    // ...
++   Sulu\Bundle\PhpcrMigrationBundle\SuluPhpcrMigrationBundle::class => ['all' => true],
+```
+
+Configure the SuluPhpcrMigrationBundle in `config/packages/sulu_phpcr_migration.yaml`:
+
+> If you are currently using Jackrabbit, use the "jackrabbit://" based DSN string.
+> After the upgrade, Apache Jackrabbit is no longer used by Sulu’s new content storage and can be removed from
+> your projects in most situations.
+
+```yaml
+# config/packages/sulu_phpcr_migration.yaml
+
+sulu_phpcr_migration:
+    # dbal://<dbalConnection>?workspace=<workspaceName>
+    # jackrabbit://<user>:<password>@<host>:<port>/server?workspace=<workspaceName>
+    #    DSN: "dbal://default?workspace=%env(PHPCR_WORKSPACE)%"
+    #    DSN: "jackrabbit://admin:admin@127.0.0.1:8080/server?workspace=%env(PHPCR_WORKSPACE)%"
+    DSN: "dbal://default?workspace=%env(PHPCR_WORKSPACE)%"
+    target:
+        dbal:
+            connection: default
+```
+
+## How to use
+
+```shell
+# Update the database to the latest migration to prepare the structure
+php bin/adminconsole doctrine:migration:migrate
```

Comment:
```suggestion
php bin/adminconsole phpcr:migrations:migrate
```

Reply:
Although I'm not really sure if we need this in the documentation in the migration bundle, as we do not have new migrations in this bundle, they are all located in the article or sulu repository and should already be part of the upgrade path in the sulu/sulu upgrade.md

WDYT?